test/parallel/test-dgram-send-address-types.js

```diff
@@ -7,13 +7,13 @@ const buf = Buffer.from('test');
 
 const onMessage = common.mustSucceed((bytes) => {
   assert.strictEqual(bytes, buf.length);
-}, 6);
+}, 4);
 
 const client = dgram.createSocket('udp4').bind(0, () => {
   const port = client.address().port;
 
   // Check valid addresses
-  [false, '', null, 0, undefined].forEach((address) => {
+  ['', null, undefined].forEach((address) => {
     client.send(buf, port, address, onMessage);
   });
```

Comment:
Non-blocking suggestion
```suggestion
const defaultCases = ['', null, undefined];
const onMessage = common.mustSucceed((bytes) => {
  assert.strictEqual(bytes, buf.length);
}, defaultCases.length);

const client = dgram.createSocket('udp4').bind(0, () => {
  const port = client.address().port;

  // Check valid addresses
  defaultCases.forEach((address) => {
    client.send(buf, port, address, onMessage);
  });
```